### DISCUSSION TITLE
Sharing model develop tools and content?

### DISCUSSION THREAD

**Discussion by serbinsh** [NGEET/fates#942] [Ideas]
During the recent NGEE Tropics All Hands call and the modeling team telecon today we continued the discussion of the need for a cloud space (Google drive, Dropbox, etc) where we can share content, parameter files, example scripts, tools, etc that aren't specific to the GitHub development space but which would be helpful to the community.  This would include a place to provide example case scripts, custom parameter files, and other tools that aren't as polished as something you would find on the FATES git or wiki.  In other words a place for work-in-progress scripts that folks are using to develop runs and that others may find useful to develop their own cases.  

This would not, for example, be files ready for the archive nor would we want the latency of posting these types of files on the archive.

Could we place a model_development sub-folder in the NGEE-Tropics Google Drive?  How would we allow external collaborators to access this information 
https://drive.google.com/drive/u/0/folders/0B_5ygYrhiTXcN2ZtMWNISVZMYWM

**Comment by glemieux** [comment]
This has come up again in the context of supporting new users via the fates docker images.  One of the big bottlenecks right now is getting users access to the inputdata to [run the host land model in docker.](https://github.com/NGEET/docker-fates-tutorial)  For context, we do not bundle the input data in the docker image due to my uncertainty with what is fair game to share and out of practicality in that it would make the already large docker images even bigger.  It would be helpful if we had a drive that we control access to that stores the necessary inputdata in lieu of getting users access to an HPC asset with the data.  My vague hope is that we might be able to pair this with a containerized version of globus that would help provide access to that data.  For reference, my current work around for is giving NGEET affiliated people access to my LBL workstation, lobata and letting them use docker and the stored data there.

@serbinsh what was the other platform that you mentioned in our containerization meeting the other day?  Was it https://osf.io/?

**Comment by glemieux** [comment]
There has been some new attempts to share scripts, parameter files, etc, on project-level basis.  I'm noting them here for reference and potential future discussion of the pros/cons of the outcomes:

- [Seed dispersal OSF.io project space](https://osf.io/k86z9/?view_only=cd066b8c81ff44eeab00bd968c004dec)
- [FATES Global Calibration Cascade Github Repo](https://github.com/adrifoster/fates-global-cal)
